docs/source/virtual_ecosystem/theory/soil/environmental_links.md

```diff
@@ -25,33 +25,54 @@ language_info:
 
 # Environmental impacts on soil processes
 
-Litter decay and soil nutrient transformations are both affected by temperature. As the
-soil model explicitly includes microbes, temperature effects many different processes in
-the model, e.g. enzymatic rates and the carbon use efficiency of microbial growth.
-Temperature is more straightforward in the litter model and just effects the decay rate
-of each litter pool. Processes that take place underground are also affected by soil
-moisture. For the soil moisture response, an empirical relationship is used for both
-litter decay and soil organic matter breakdown.
+Litter decay and soil nutrient transformations are both affected by environment. At the
+most basic level these are impacts on the microbial components of the soil and litter
+models, which then impact the models more broadly. There are three different ways that
+these impacts are represented in the models. The rates of processes that are implicitly
+driven by microbes can change, the growth rates of the different microbial groups can be
+directly affected, or the enzymatic rates can be affected. Each of these cases will be
+dealt with in detail below.
 
-:::{admonition} In progress 🛠️
+At present, the only environmental impact we represent that isn't mediated by microbes
+is the the rate at which nutrients leach from the soil. As such, this process does not
+fit into any of the cases mentioned already, and so it will discuss in a separate
+section.
 
-The representation of soil microbial communities has still not been finalised. Once this
-has happened this section of the documentation will be extended to detail the relevant
-thermal responses.
+## Environmental modification of implicit rates
```

Comment:
Can this subheading be removed? It's very similar to the title.

Reply:
I think a subheading is needed so that the section can be linked to elsewhere in the docs, but agree that that phrasing is repetitive. I've rewritten the subheading so that it is more distinct